### DISCUSSION TITLE
Change the default configuration of fields and columns

### DISCUSSION THREAD

**Discussion by mjood10tn** [filamentphp/filament#6458] [Help] [answered]
Is there a way to change the default configuration of fields and columns?

For example, I want all  DatePickers to start on Sunday, without the need to use the weekStartsOnSunday() function every time I use a DatePicker.
.
.
Thank you

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/forms/fields#global-settings

**Reply by mjood10tn** [reply]
Thank you
